SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
+. Implementation Conformance Statement (ICS) tables from each of these PKP standards is included in TF-1 Appendix B of this specification, with indication as to whether, how and where each requirement is addressed
+. "Safety, Effectiveness, Security Considerations and Requirements" sections are integrated throughout the profile specifications to link from the PKP ICS table requirements to the satisfying capabilities.
+[none]
+.. Additional non-PKP risk management will also be performed by subject matter experts and formalized in these SES consideration sections, where appropriate.
+
+These SES Considerations sections are the result of the IHE "Security Considerations" sections + the IHE Devices "Safety Considerations" sections, but now consolidated into a single SES Considerations section that includes the 3rd risk management property, Effectiveness, and is tied to meeting the requirements of specific standards.
+
+NOTE:  The moniker *_SES+MDI_* is shorthand to refer to the integration of the technical medical device interoperability (MDI) specifications  with the application of quality / risk management SES standards and processes.
+
+How does this address the "interoperability trust gap"?  By integrating SES directly into the specifications, especially integrating the ISO/IEC 11073-1070x standards, enabling "plug-and-trust" system product components, the SES implementation and operational requirements and responsibilities are greatly reduced, the "gap" is filled for all stakeholders, and the the goals of improved safety, security and clinical effectiveness of technology can be readily realized.
+
+=== SDPi Requirements Modeling & Integration
+
+As pointed out above, requirements interoperability (RI) based on robust model-based metadata is a core, innovative aspect of this SDPi profiles specification.  Given the ultimate intent to realize this description as a _Model Centric (MC) single-source-of-truth, computable, simulatable, verifiable and validatable system of systems interoperability specification_, and recognizing that it will take a significant transition period from a document-centric approach to a model-centric approach, a simplified requirements model is provided below but is aligned with the <<omg_sysml_2-0_spec>> section 7.20 Requirements language.  Of course, that specification provides for significantly more detailed and complext modeling, the general constructs may be used in this document to start the transition toward that model.  Note that SysML 2.0 also better supports model interoperability (tool-independent model exchange) and _Model-based Systems Engineering (MBSE)_ (see https://en.wikipedia.org/wiki/Model-based_systems_engineering[MBSE Wikipedia article and references]), as well as the <<ihe_eu_experience_2021_presentation_cooper_schlichting>> for an overview presentation of MBSE, MedTech system V&V, and IHE Conformity Assessment.
```

Comment:
```suggestion
As pointed out above, requirements interoperability (RI) based on robust model-based metadata is a core, innovative aspect of this SDPi profiles specification.  Given the ultimate intent to realize this description as a _Model Centric (MC) single-source-of-truth, computable, simulatable, verifiable and validatable system of systems interoperability specification_, and recognizing that it will take a significant transition period from a document-centric approach to a model-centric approach, a simplified requirements model is provided below but is aligned with the <<omg_sysml_2_0_spec>> section 7.20 Requirements language.  Of course, that specification provides for significantly more detailed and complex modeling, the general constructs may be used in this document to start the transition toward that model.  Note that SysML 2.0 also better supports model interoperability (tool-independent model exchange) and _Model-based Systems Engineering (MBSE)_ (see https://en.wikipedia.org/wiki/Model-based_systems_engineering[MBSE Wikipedia article and references]), as well as the <<ihe_eu_experience_2021_presentation_cooper_schlichting>> for an overview presentation of MBSE, MedTech system V&V, and IHE Conformity Assessment.
```

Reply:
A general rule of thumb for those text-based and Git-based document development: 1 sentence per line increases readability in the text editor und display of diffs. 

```
As pointed out above, requirements interoperability (RI) based on robust model-based metadata is a core, innovative aspect of this SDPi profiles specification.
Given the ultimate intent to realize this description as a _Model Centric (MC) single-source-of-truth, computable, simulatable, verifiable and validatable system of systems interoperability specification_, and recognizing that it will take a significant transition period from a document-centric approach to a model-centric approach, a simplified requirements model is provided below but is aligned with the <<omg_sysml_2_0_spec>> section 7.20 Requirements language.
Of course, that specification provides for significantly more detailed and complex modeling, the general constructs may be used in this document to start the transition toward that model.
Note that SysML 2.0 also better supports model interoperability (tool-independent model exchange) and _Model-based Systems Engineering (MBSE)_ (see https://en.wikipedia.org/wiki/Model-based_systems_engineering[MBSE Wikipedia article and references]), as well as the <<ihe_eu_experience_2021_presentation_cooper_schlichting>> for an overview presentation of MBSE, MedTech system V&V, and IHE Conformity Assessment.
```

Single line breaks will not create a line break in the output format.

Reply:
Updated